example/libclang-example/generated_bindings.dart

```diff
@@ -7928,64 +7928,8 @@ abstract class CXGlobalOptFlags {
 /// Uniquely identifies a CXFile, that refers to the same underlying file,
 /// across an indexing session.
 class CXFileUniqueID extends ffi.Struct {
-  @ffi.Uint64()
-  external int _unique_data_item_0;
-  @ffi.Uint64()
-  external int _unique_data_item_1;
-  @ffi.Uint64()
-  external int _unique_data_item_2;
-
-  /// Helper for array `data`.
-  ArrayHelper_CXFileUniqueID_data_level0 get data =>
-      ArrayHelper_CXFileUniqueID_data_level0(this, [3], 0, 0);
-}
-
-/// Helper for array `data` in struct `CXFileUniqueID`.
-class ArrayHelper_CXFileUniqueID_data_level0 {
-  final CXFileUniqueID _struct;
-  final List<int> dimensions;
-  final int level;
-  final int _absoluteIndex;
-  int get length => dimensions[level];
-  ArrayHelper_CXFileUniqueID_data_level0(
-      this._struct, this.dimensions, this.level, this._absoluteIndex);
-  void _checkBounds(int index) {
-    if (index >= length || index < 0) {
-      throw RangeError(
-          'Dimension $level: index not in range 0..$length exclusive.');
-    }
-  }
-
-  int operator [](int index) {
-    _checkBounds(index);
-    switch (_absoluteIndex + index) {
-      case 0:
-        return _struct._unique_data_item_0;
-      case 1:
-        return _struct._unique_data_item_1;
-      case 2:
-        return _struct._unique_data_item_2;
-      default:
-        throw Exception('Invalid Array Helper generated.');
-    }
-  }
-
-  void operator []=(int index, int value) {
-    _checkBounds(index);
-    switch (_absoluteIndex + index) {
-      case 0:
-        _struct._unique_data_item_0 = value;
-        break;
-      case 1:
-        _struct._unique_data_item_1 = value;
-        break;
-      case 2:
-        _struct._unique_data_item_2 = value;
-        break;
-      default:
-        throw Exception('Invalid Array Helper generated.');
-    }
-  }
+  @ffi.Array.multi([3])
+  external ffi.Array<ffi.Uint64> data;
```

Comment:
Since this also provides `[]` and `[]=` this is actually not an API change at all, that is pretty sweet!